src/payloads/InterviewDates.ts

```diff
@@ -0,0 +1,6 @@
+import { IsArray } from 'class-validator';
+
+export class InterviewDatesResponse {
+  @IsArray()
+  interview: Date[];
```

Comment:
We should try to have payloads as strings - Date gets converted into ISO string in json anyways
Better to be explicit here